### ISSUE TITLE
Inject node doesn't work

### ISSUE THREAD

**Issue by S474N** [node-red/node-red#4670]
### Current Behavior

First I have dead loop in log, NR restarted. Last thing in log is:
`23 Apr 19:29:44 - [error] UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason #<Object> .`

I suspected the node-red-contrib-home-assistant-websocket plugin, when I delete it, Nodered does start, I can connect to the web administration, but all the inject nodes are gray and cannot be started.

I tested downgrading the docker image or even changing it because of nodejs. I also tried restoring the data from the previous day, but all to no avail.

During testing, I also turned off HA, which runs on another HW, to exclude influence.

I also tried updating npm to the latest version, I tried "audit fix".

Now I don't know what else to try.

### Expected Behavior

Working inject nodes.

### Steps To Reproduce

Don't know

### Example flow

```
paste your flow here
```

### Environment

- Node-RED version: v3.1.9, v3.1.8 - docker version
- Node.js version: 14, 16, 18
- npm version: that are in official docker images 
- Platform/OS:  arm64/Raspbian Bookworm
- Browser: Chrome

**Comment by S474N** [comment]
Log from `npm audit fix --force`

https://pastebin.com/wmrssAkQ

**Comment by hardillb** [comment]
Running `npm audit --fix` in the docker container isn't going to change anything as the changes will be reverted when the container is restarted 

The inject buttons being disabled has been seen occasionally and can sometimes be down to the browser caching old versions of the frontend is files. Please ensure the browser cache is cleared

**Comment by S474N** [comment]
Already tested - also new private window, in EDGE, same problem.

**Comment by S474N** [comment]
Don't know, what is really wrong:
![image](https://github.com/node-red/node-red/assets/16512337/33367927-f143-4dfb-9763-aadadf349d40)

**Comment by knolleary** [comment]
What version of node.js are you using? That suggests you are on an old version that doesn't support the Object.hasOwn api

**Comment by S474N** [comment]
Yes you're right. In desperation, I tried all versions, upgrade/downgrade, until I ended up on node.js v14, where at least the basic functioning of my home works.

I am convinced that the error is in my old installation and in a lot of unresolved dependencies right within npm. Unfortunately, I didn't have time to completely reinstall everything, including all plugins.

**Comment by knolleary** [comment]
Closing as part of a tidy-up of the issue tracker — this has been inactive for a long time and much has changed in Node-RED since. If it's still relevant on a current version, please open a fresh issue with up-to-date details and we'll take a look. Thanks for raising it.
